augur/clades.py

```diff
@@ -102,12 +102,9 @@ def assign_clades(clade_designations, all_muts, tree, ref=None):
     '''
 
     clade_membership = {}
+    basal_clade_nodes = {}
```

Comment:
This would be a great place for a block comment describing what `clade_membership` and `basal_clade_nodes` represent, to set the reader's expectations for the rest of the code in this function.

Reply:
Done 👍